### PR TITLE
RM-194727 Release json_schema 5.0.0-rc2

### PR DESCRIPTION
**Pull request by rmconsole-readonly-wk**
Pull Requests included in release:
* Patch changes:
	* [CUI-1729: Updated changelog for version 5.0.0-rc1.](https://github.com/Workiva/json_schema/pull/154)

Requested by: @kyleferguson-wf

Diff Between Last Tag and Proposed Release: https://github.com/Workiva/json_schema/compare/5.0.0-rc1...Workiva:release_json_schema_5.0.0-rc2
Diff Between Last Tag and New Tag: https://github.com/Workiva/json_schema/compare/5.0.0-rc1...5.0.0-rc2

The logs for the request that created this PR can be found [here](https://w-rmconsole.appspot.com/release/release_event/5466858621239296/logs/)
This pull request can be recreated by clicking [here](https://w-rmconsole.appspot.com/api/v2/rosie/reTriggerReleaseEvents/5466858621239296/?repo_name=Workiva%2Fjson_schema&pull_number=155)

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Comment by kyleferguson-wk** [comment]
+1

File: pubspec.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: json_schema
-version: 5.0.0-rc1
+version: 5.0.0-rc2rc1
```
Comment (kyleferguson-wk) [inline pubspec.yaml:2]: wait this actually does not look correct, this should just be `5.0.0-rc2`

Comment (kyleferguson-wk) [reply pubspec.yaml:2]: placed hold until I can confirm whether this is expected update or not

Comment (kyleferguson-wk) [reply pubspec.yaml]: Confirmed this needs to be `5.0.0-rc2`, pushed update to fix 👍

**Review by joshriesenbach-wk — APPROVED** [review]
+1

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**